ddl/cache.lua

```diff
@@ -0,0 +1,99 @@
+local fiber = require('fiber')
+
+local cache = {
+    spaces = {},
```

Comment:
I think it would be useful to describe a structure of cache for every record in comment or as a table with `nil` as values.
For example:
```lua
space = {
    'space_name' = {
        'raw' = nil, -- raw sharding metadata, used for ddl.get().
        'processed' = nil -- sharding function that is ready to call or a string with error.
    }
}
```

Reply:
fixed